integration-tests/src/test/resources/compose/docker-compose-solr.yml

```diff
@@ -17,6 +17,7 @@ services:
       - DEBUG=true
       - JMX_ENABLED=true
     ports:
+      - "8444:8443"
```

Comment:
fixed port on a CI/CD is a bad idea

Reply:
As discussed, this was to run tests locally on Windows machines. Doesn't impact Linux machines, nor github actions.